publish/publish.proj

```diff
@@ -152,6 +152,19 @@
     <Exec Condition="'@(_DownloadedSymbolsPackages)' != ''" Command="$(NuGetPushSymbolsCommand) %(_DownloadedSymbolsPackages.Identity)" />
   </Target>
   
+  <Target Name="PublishDebToolPackageToFeed"
+          Condition="'$(PublishDebToolToFeed)' == 'true'">
+    <Error Condition="'$(CliNuGetFeedUrl)' ==''" Text="Missing required property CliNuGetFeedUrl" />
+    <Error Condition="'$(CliNuGetApiKey)' == ''" Text="Missing required property CliNuGetApiKey" />
+    <ItemGroup>
+      <DebToolPackages Include="$(PackagesOutDir)intermediate/*.nupkg" />
```

Comment:
Using the `intermediate` folder here seems less than ideal.  I don't really like that we are producing the deb-tool nupkg here and then copying it out.

What if instead, we said:

`<DebToolPackages Include="$(PackagesOutDir)/dotnet-deb-tool.*.nupkg" />`

This is similar logic to what we are doing here:

[https://github.com/dotnet/core-setup/blob/master/src/pkg/packaging/deb/package.targets#L18-L23](https://github.com/dotnet/core-setup/blob/master/src/pkg/packaging/deb/package.targets#L18-L23)